src/server/README.md

```diff
@@ -479,22 +479,34 @@ Faculty Member API
                 -   Unprocessable Entity -> Status code: 422
                 -   Internal server error -> Status code: 500
 
-    -   api/facultyMember/get-applicants/:jobId
+    -   api/facultyMember/get-applicants
 
         -   Returns a list of students who applied for a job with the given job Id from the database.
         -   Body: None
         -   Authorization restrictions:
             -   User must be logged in
             -   User must be a faculty member
             -   User must be an owner of the job
-        -   Parameters: None
+        -   Parameters:
+            ```
+            {
+                jobId: number;
+                departmentIds: string[];
+                classStandings: 'freshman' | 'sophomore' | 'junior' | 'senior'[];
+                minimumGpa: number;
```

Comment:
I think minimumGpa will be a string too since it comes in the query parameters

Reply:
Done.